test/functional/p2p_blocksonly.py

```diff
@@ -9,7 +9,6 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import assert_equal
 
-
```

Comment:
I think pepe-8 wants two lines here

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/19963/commits/ee11320b0616ae6811a0aef6a905e62e205a144f